ci/test_size_regression.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Checks the absolute size, and the relative increase of a file.
+MAX_SIZE=4000000
+MAX_PERC=1
```

Comment:
This might be too strict right now with so much in flux. Can we run the check but not have it block PRs?

Reply:
I think I'd rather change the constants to laxer terms if you want, but I would like this to block because otherwise we won't pay attention.